cylc/flow/scripts/cylc.py

```diff
@@ -46,32 +46,24 @@ def get_version(long=False):
 
 
 USAGE = f"""{cylc_header()}
-Cylc ("silk") orchestrates complex cycling (and non-cycling) workflows.
+Cylc ("silk") efficiently manages distributed cycling workflows.
+This is Open Source software (GPL-3.0): see "cylc help license".
 
 Version:
-  $ cylc version --long           # print cylc-flow version and install path
+  $ cylc version --long
   {get_version(True)}
 
 Usage:
-  $ cylc help license             # view the Cylc license (GPL-3.0)
-  $ cylc help all                 # list all commands
-  $ cylc validate <workflow>      # validate a workflow configuration
-  $ cylc install <workflow>       # install a workflow
-  $ cylc play <workflow>          # run/resume a workflow
-  $ cylc scan                     # list all running workflows (by default)
-  $ cylc tui <workflow>           # view/control workflows in the terminal
-  $ cylc stop <workflow>          # stop a running workflow
-
-Command Abbreviation:
-  # Commands can be abbreviated as long as there is no ambiguity in
-  # the abbreviated command:
-  $ cylc trigger WORKFLOW//CYCLE/TASK    # trigger TASK in WORKFLOW
-  $ cylc trig WORKFLOW//CYCLE/TASK       # ditto
-  $ cylc tr WORKFLOW//CYCLE/TASK         # ditto
-  $ cylc t                               # Error: ambiguous command
+  $ cylc <sub-command> <OPTS> <ARGS>
+
+Selected sub-commands are listed below. To view ALL sub-commands:
+  $ cylc help all
+
+To view sub-command help:
+  $ cylc <sub-command> --help
 
 Cylc IDs:
-  Cylc IDs take the form:
+  Workflows and tasks are uniquely identified by IDs of the form:
```

Comment:
```suggestion
  Workflows and tasks are identified by IDs of the form:
```

(subjective: just trying to reduce the word count)